test/test-classTetrisMoves.js

```diff
@@ -0,0 +1,301 @@
+var sinon = require('sinon');
+var assert = require('chai').assert;
+var tetris = require('../tetris');
+describe("All moves tetromino", function () {
+    let stubForGetRandomTypeOfTetromino = sinon.stub(tetris, "getRandomTypeOfTetromino");
+    it("Mini Game", function () {
+        stubForGetRandomTypeOfTetromino.returns("O")
+        let fieldSize = { x: 8, y: 6 }
+        let elementTetris = new tetris.Tetris(fieldSize)
```

Comment:
why it's called elementTetris? It's not element in any way.
Just call it game or something